### DISCUSSION TITLE
slide/drag to multiselect

### DISCUSSION THREAD

**Discussion by badmannen** [immich-app/immich#4935] [Feature Request]
### The feature

It would be nice ( if the Android API  supports it), to have similar to IOS to "multi-select" items / photos by dragging over a row of pictures instead of "clicking" them one by one.

would speed up selections immensely.

similar to this example:
![dJtMK](https://github.com/immich-app/immich/assets/17902962/7e5a7059-0478-4554-b094-9be52f50d16a)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bouncyninja** [comment]
This would be a great QOL improvement.  This is something that the app "that shall not be named" has and allows for lots of pictures to be added to albums in more rapid and less cumbersome manner.  I am using IOS and contrary to OP's post, it is not something that seems to be working for me.

**Comment by aisbergde** [comment]
Improving multiple select would also be required in the Web GUI. At least the common usage of shift+click to extend the selection is missing. Should this be a separate issue?

**Reply by bo0tzz** [reply]
Shift+click for multiselect in the web UI is definitely supported.

**Reply by aisbergde** [reply]
you are partially right - it is supported only in the timeline view.
It is not implemented in the search result view.
Should I create a separate feature request, or is it OK here to write "improve multi select in search result view, enable the usage of shift-mouse"?

**Reply by bo0tzz** [reply]
Some of the secondary views are still using an old gallery implementation. I believe there are already issues/discussions open for refactoring those.

**Reply by aisbergde** [reply]
can't find, created #6162

**Comment by bo0tzz** [comment]
Duplicate of #3999
